users.py

```diff
@@ -45,52 +50,54 @@ def create_user():
         # psycopg2 sql injection
         cursor, conn = get_db_connection()
         if cursor is None or conn is None:
-            return jsonify({'error' : 'Failed to connect to server' }), 500
+            return jsonify({'error' : 'Internal Error' }), 500
         
-        # Checking if a user with this email already exists
-        cursor.execute("SELECT id FROM users WHERE email = %s", (email,))
-        existing_user = cursor.fetchone()
-        if existing_user:
-            cursor.close()
-            conn.close()
-            return jsonify({'error' : 'User with this email already exist!'}), 409
-
         cursor.execute("""
             INSERT INTO users (id, first_name, last_name, email, password_hash, created_at, updated_at) 
             VALUES (%s, %s, %s, %s, %s, %s, %s) RETURNING id;
         """, (user_id, data['first_name'], data['last_name'], email, hashed_password, str(created_at), str(updated_at)))
+        
         conn.commit() 
```

Comment:
Check the "auto_commit" feature of the library you're using. Will save you some time doing this each time.